tools/copy_inputs.sh

```diff
@@ -170,22 +208,31 @@ if [ -f $dfile ]; then
 fi
 
 # copy config script for execution
-config_script=$DA_EXPDIR_SRC/$src/config.sh
+config_script=$DA_EXPDIR/$src/config.sh
 if [ -f $config_script ] ; then
   cp $config_script $DA_EXPDIR/$tar
-else
-  echo 'Skipping config file (no config.sh file found).'
+fi
+
+# copy STL for execution
+stlfile=$(find $DA_EXPDIR/$src -iname "*.stl")
+if [ -f "$stlfile" ] ; then
+  cp "$stlfile" $DA_EXPDIR/$tar
 fi
 
 # copy *.txt files for IBM implementation
-  cp $DA_EXPDIR_SRC/$src/*.txt $DA_EXPDIR/$tar
+IBM_files=$DA_EXPDIR/$src/*.txt
```

Comment:
Let's not use only wildcard here - there may be other .txt files in the directory that the user should copy themselves if they want to. Instead do something like `IBM_files=$DA_EXPDIR/$src/solid_?.txt` and similar for `fluid_boundary_?.txt` and `facet_sections_?.txt`. Also probably should still copy `info.txt` just in case.

Reply:
Agreed and done!